app/dashboard/static/js/app/view-tests-all.2018.9.js

```diff
@@ -28,251 +31,134 @@ require([
+                operation_id: 'status-' + result._id.$oid,
                 resource: 'count',
-                document: 'test_group',
-                query: query
+                document: 'test_regression',
+                query: qStr,
             });
         }
 
-        if (response.length > 0) {
-            batchOps = [];
-            response.forEach(createBatchOp);
-
-            deferred = request.post(
-                '/_ajax/batch', JSON.stringify({batch: batchOps}));
+        batchOps = [];
+        results.forEach(createBatchOp);
+        deferred = request.post(
+            '/_ajax/batch', JSON.stringify({batch: batchOps}));
 
-            $.when(deferred)
-                .fail(error.error, getGroupsCountFail)
-                .done(getGroupsCountDone);
-        }
+        $.when(deferred)
+            .fail(error.error, getBatchStatusFailed)
+            .done(getBatchStatusDone)
     }
 
-    function getTestsDone(response) {
+    function updateTestTable(response) {
         var columns;
+        function _renderPlan(data, type, object) {
+            return ttest.renderKernel(data, type, 
+                '/test/job/'+object.job+
+                '/branch/'+object.git_branch+
+                '/kernel/'+object.kernel+
+                '/plan/'+data);
+        }
 
-        // Internal wrapper to provide the href.
-        function _renderDetails(data, type) {
-            return ttest.renderDetails('/test/board/' + data + '/', type);
+        function _renderTree(data, type) {
+            return ttest.renderTree(
+                data, type, '/job/' + data + '/');
         }
 
-        // Internal wrapper to provide the oreder count.
-        function _renderGroupsCount(data, type) {
-            var rendered;
+        function _renderKernel(data, type, object) {
+            var href = '/test/job/';
+            href += object.job;
+            href += '/branch/';
+            href += object.git_branch;
+            href += '/kernel/';
+            href += data;
+            href += '/';
+            return ttest.renderKernel(data, type, href);
+        }
 
-            rendered = null;
+        function _renderBranch(data, type, object) {
+            var rendered;
+            var aNode;
+            var href;
+            var tooltipNode;
+            
+            rendered = data;
             if (type === 'display') {
-                rendered = ttest.countBadge({
-                    data: data,
-                    type: 'default',
-                    idStart: 'groups-',
-                    extraClasses: ['groups-count-badge']
-                });
-            } else if (type === 'sort') {
-                if (gGroupsCount.hasOwnProperty('groups-count-' + data)) {
-                    rendered = gGroupsCount['groups-count-' + data];
+                tooltipNode = html.tooltip();
+                tooltipNode.setAttribute('title', data);
+                if (href && href.length > 0) {
```

Comment:
There is no `href` in this case, it looks like this was pasted from somewhere else.  At the moment, these links are not being created.  The `_renderBranch()` method should be very similar to the `_renderKernel()` one, with just the branch name added to the URL.

Reply:
@gctucker I updated `_renderBranch` to use `renderKernel` and avoid to duplicate code in cases like that, is it ok? I think we can update the code soon to make this kind of function more "generic", so it will be ok use in those cases.